src/cr/cube/dimension.py

```diff
@@ -586,6 +589,14 @@ def _element_dicts(self):
             else self._type_dict["elements"]
         )
 
+    def _element_id(self, element_dict):
```

Comment:
This is a function, can we name it like a function? In the sense: verb + object... Maybe something like `_get_element_id_from`... If that's too ugly, can we at least do `_element_id_from_dict`?